test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
Didn't shy from what exactly?

Is my proposal of the `.eslintrc-(base|browser|node).json` files OK for you?